contracts/pair/src/storage.rs

```diff
@@ -336,18 +336,19 @@ mod tests {
         assert_eq!(result, Ok((100, 200)));
     }
 
+    #[ignore]
```

Comment:
It's a good practice to always comment why some test is ignored, even if it's just for a short time (you never know how short that time will be...).
Just add something like `Currently ignored because it's difficult to find proper values for test`.